pkg/apis/work/v1alpha2/binding_types.go

```diff
@@ -206,6 +217,12 @@ type GracefulEvictionTask struct {
 	// +required
 	Producer string `json:"producer"`
 
+	// DeleteOption represents that how to deal with the legacy GracefulEvictionTask.
+	// Defaults to "Graciously".
+	// +kubebuilder:default=Graciously
+	// +optional
+	DeleteOption DeleteOption `json:"deleteOption,omitempty"`
```

Comment:
```golang
	// GracePeriodSeconds is the duration in seconds before the item should be
	// deleted. The value must be a non-negative integer.
	// The value zero indicates delete immediately.
	// It can not co-exist with SuppressDeletion.
	// +optional
	GracePeriodSeconds *int64 `json:"gracePeriodSeconds,omitempty"`

	// SuppressDeletion represents the grace period that will be persistent until
	// the tools or human intervention stops it.
	// It can not co-exist with GracePeriodSeconds.
	SuppressDeletion *bool `json:"suppressDeletion,omitempty"`
```